jupyterhub/tests/browser/test_browser.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the Playwright Python"""
+
+import re
+
+import pytest
+from playwright.async_api import expect
+
+from jupyterhub.tests.utils import public_host
+from jupyterhub.utils import url_path_join
+
+pytestmark = pytest.mark.playwright
+
+
+async def login(browser, username, password):
+    """filling the login form by user and pass_w parameters and iniate the login"""
+
+    await browser.get_by_label("Username:").click()
+    await browser.get_by_label("Username:").fill(username)
+    await browser.get_by_label("Password:").click()
+    await browser.get_by_label("Password:").fill(password)
+    await browser.get_by_role("button", name="Sign in").click()
+
+
+async def test_open_login_page(app, browser):
```

Comment:
For migration, let's _remove_ each selenium test we add here, so that it's easier to keep track of the migration progress.

Reply:
this simple case is absent in the selenium scope, but I will follow your approach with removing in the next PR, it is a good point